src/js/models/AppModel.js

```diff
@@ -1079,6 +1079,15 @@ define(['jquery', 'underscore', 'backbone'],
       */
       isJSONLDEnabled: true,
 
+      /**
+      * If true, users can see a "Publish" button in the MetadataView on all non-public datasets regardless if
+      * a DOI is assigned to the dataset
+      * If false, the default behavior would take place based on the {@link AppConfig#enablePublishDOI} config.
+      * @type {boolean}
+      * @default false
```

Comment:
When this gets released, I update all the @since x.x.x with the new version.
```suggestion
      * @default false
      * @since x.x.x
```

Reply:
That's perfect @helbashandy, thank you! In the future, just use `x.x.x` as the version - these are all replaced with the actual version number at the time of release - that way there's flexibility if something has to get pushed to a different release, or the version # changes for any reason.